src/cmd-buildextend-live

```diff
@@ -91,42 +95,69 @@ name_version = f'{base_name}-{args.build}'
 # to shorten this more intelligently, otherwise we truncate the
 # version which may impede uniqueness.
 volid = name_version[0:32]
+build_path = Path(f"./{builddir}/{base_name}-{args.build}").resolve()
```

Comment:
Minor: since we're using other `os.path` APIs already, this could just be `os.path.abspath()` and would avoid the `pathlib` import.

We shouldn't need the `./` prefix.